LibreNMS/IPMI/IPMIClient.php

```diff
@@ -26,13 +26,15 @@
 namespace LibreNMS\IPMI;
 
 use ErrorException;
-use LibreNMS\Config;
+use Illuminate\Support\Facades\Cache;
 
 /**
  * Represents an IPMI connection with a host machine.
  */
 class IPMIClient
 {
+    const SDR_TTL = 60 * 15;
```

Comment:
Should this be a config parameter?

Reply:
Try to minimize config variables in general.  They are easy to add later if needed.